src/Microsoft.ML.Core/Prediction/ITrainer.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.ML
     [BestFriend]
     internal delegate void SignatureMultiOutputRegressorTrainer();
     [BestFriend]
-    internal delegate void SignatureRankerTrainer();
+    internal delegate void SignatureRankingTrainer();
```

Comment:
>SignatureRankingTrainer [](start = 27, length = 23)

Since this is internal it is not a big deal, but note that this is incorrect. So you see how it is `SignatureBinaryClassifierTrainer` above? This is because the thing that does the *task* binary classificat*ion* is a binary classif*ier*. So this should have been retained as signature *ranker* trainer. What is the task? Ranking. What is the thing doing the task? A ranker.

Sort of like how things are being transformer, and the thing that does this is called a transformer.

Not a huge deal, but if you have to post another iteration anyway, this is what I would call a hypercorrection.